infrastructure/ansible/roles/dataset_loader/defaults/main.yml

```diff
@@ -71,14 +71,14 @@ dl_ds_default_cdns:
     dnssecEnabled: false
 
 # TO Types
-dl_ds_merged_types: "{{ dl_ds_default_types }}"
-dl_ds_default_types:
-  - name: TR_LOC
-    description: Traffic Router Logical Location
-    useInTable: cachegroup
-  - name: INFRA_LOC
-    description: Generic Infrastructure Logical Location
-    useInTable: cachegroup
+#dl_ds_merged_types: "{{ dl_ds_default_types }}"
+#dl_ds_default_types:
+#  - name: TR_LOC
+#    description: Traffic Router Logical Location
+#    useInTable: cachegroup
+#  - name: INFRA_LOC
+#    description: Generic Infrastructure Logical Location
+#    useInTable: cachegroup
```

Comment:
These should still be included and used instead of an undefined `private types` variable.  Just make default types an empty array.

Reply:
updated